app/Http/Requests/Movie/CreateRequest.php

```diff
@@ -27,6 +27,8 @@ public function rules()
             'title' => ['required', 'min:1', 'max:255'],
             'year' => ['required', 'integer'],
             'description' => ['required', 'min:100'],
+            'genres' => ['required', 'array', 'min:1'],
+            'actors' => ['required', 'array', 'min:1'],
```

Comment:
давай добавим проверку, что нам существущие айди актеров и жанров отправляют
по аналогии как делали на занятиях 

https://github.com/akarashchuk/laravel-study/blob/master/app/Http/Requests/Article/EditRequest.php#L32
```suggestion
            'genres' => ['required', 'array', 'min:1'],
            'genres' => ['required', 'exists:genres,id'],
            'actors' => ['required', 'array', 'min:1'],
            'genres' => ['required', 'exists:actors,id'],
```

и в редактировании тоже поправь